### ISSUE TITLE
Having multiple locations for basePath

### ISSUE THREAD

**Issue by seanlowe** [ridafkih/schemix#61]
Hey, building a fullstack application that uses schemix and I wanted to see if there was a way to allow multiple directory locations for src files. I want to make my app extensible and provide clear locations for those extensions to live so anyone who may decide to contribute doesn't have to deal with adding code in my base models.

Maybe something like :
```javascript
createSchema({
  basePath: [__dirname, 'somepath/to/models`],
  // other options
})
```

or possibly:
```javascript
createSchema({
  basePath: __dirname,
  additionalPaths: ['somepath/to/models', 'secondpath/to/models']
  // other options
})
```

Let me know if you need/want any additional information or if this is just plain too hard to do. Thanks!

**Comment by seanlowe** [comment]
I looked through the source code a little bit. I think it should be fairly simple to add this. I'm looking at the PrismaSchema.ts > toString() function.

Existing code:
```typescript
if (this.basePath) {
  await importAllFiles(this.basePath, "enums");
  await importAllFiles(this.basePath, "models");
  await importAllFiles(this.basePath, "mixins");
}
```

Possible update:
```typescript

constructor(
  // constructor stuff
  private readonly additionalPaths?: string[]
) {}

// other functions and stuff

public toString(): Promise<string> {
  return new Promise(async (resolve) => {
    const allPaths = [this.basePath, ...this.additionalPaths];

    allPaths.forEach((path) => {
      await importAllFiles(path, "enums");
      await importAllFiles(path, "models");
      await importAllFiles(path, "mixins");
    }

    // rest of function
```

I can put up a PR for this if you want.

**Comment by ridafkih** [comment]
I think the `additionalPaths` route is the way to go, should be easy enough to implement. @seanlowe have you given it a go yet?

**Comment by seanlowe** [comment]
I haven’t actually tried. I’ll try to get one up today.

**Comment by seanlowe** [comment]
@ridafkih I have created a PR for this issue when you have a chance to look at it. Let me know what you think.

**Comment by ridafkih** [comment]
Closing, #66 has been released in v1.12.0.

Thanks for the suggestion and implementation!
